tests/test_cli.py

```diff
@@ -0,0 +1,336 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Tests for `mlbench_core.cli` package."""
+
+import pytest
+from mlbench_core.cli import cli_group
+from google.auth.exceptions import DefaultCredentialsError
+
+# import click
+
+create_gcloud = cli_group.commands["create-cluster"].commands["gcloud"].main
```

Comment:
Click has a `CliRunner` that would allow you to just import the base group and use the runner to invoke commands: https://click.palletsprojects.com/en/7.x/testing/ might make testing easier